server/graphql/v1/queries.js

```diff
@@ -317,24 +317,50 @@ const queries = {
       dateTo: { type: GraphQLString },
       /** @deprecated since 2018-11-29: Virtual cards now included by default when necessary */
       includeVirtualCards: { type: GraphQLBoolean },
+      fetchDataFromLedger: { type: GraphQLBoolean }, // flag to go with either api or ledger transactions
+      includeHostedCollectivesTransactions: {
+        type: GraphQLBoolean,
+      } /** flag to determine
+        whether we should include the transactions of the collectives of that host(if it's a host collective) */,
     },
     async resolve(_, args) {
+      let fetchDataFromLedger = process.env.GET_TRANSACTIONS_FROM_LEDGER || false;
+      if (args.fetchDataFromLedger) {
```

Comment:
This won't go in the if when `args.fetchDataFromLedger` is false.

Reply:
the default will be false which makes it ok as it'll only catch the `if` if the args are true, otherwise it will already be false **OR** defined by `process.env.GET_TRANSACTIONS_FROM_LEDGER`.

The way it'll be is:
1. the default is false
2. if there is `process.env.GET_TRANSACTIONS_FROM_LEDGER` it will go with that.
3. if there is `args.fetchDataFromLedger` it'll go with that regardless the default and `process.env.GET_TRANSACTIONS_FROM_LEDGER`.

Reply:
No, `if (args.fetchDataFromLedger) {`  is not checking if `args.fetchDataFromLedger` is `undefined` or not. It's checking if `args.fetchDataFromLedger` is truthy or falsy. If the value is `false`, it's not going in the condition.

If `GET_TRANSACTIONS_FROM_LEDGER` is true and `args.fetchDataFromLedger` is false, we're not having the expected behavior.